resources/js/Pages/Cart/CheckoutComponent.vue

```diff
@@ -0,0 +1,128 @@
+                                                <td class="align-middle col-2">
+                                                    <input v-model="item.quantity" type="number" class="form-control" @change="update(item)">
+                                                </td>
+                                                <td class="align-middle col-2">₱ {{ item.price }}</td>
+                                                <td class="align-middle text-right  col-2">₱ {{ item.subtotal }}</td>
+                                            </tr>
+                                        </tbody>
+                                        <tfoot>
+                                            <tr>
+                                                <td class="text-right font-weight-bold" colspan="4">Total</td>
+                                                <td class="text-right">₱ {{ total }}</td>
+                                            </tr>
+                                        </tfoot>
+                                    </table>
+                                </div>
+                            </div>
+                            <div class="row card">
+                                <div class="card-header p-2 bg-dark text-center">
+                                    <h5>Personal Information</h5>
+                                </div>
+                                <div class="card-body">
+                                    <!-- <div class="row">
+                                        <div class="form-group col-6">
+                                            <label for="firstName">First Name</label>
+                                            <input v-model="form.firstname" type="email" class="form-control" id="firstName" placeholder="Enter First Name">
+                                        </div>
+                                        <div class="form-group col-6">
+                                            <label for="exampleInputEmail1">Last Name</label>
+                                            <input v-model="form.lastname" type="email" class="form-control" id="exampleInputEmail1" placeholder="Enter Last Name">
+                                        </div>
+                                    </div> -->
+                                    <div class="form-group">
+                                        <label for="exampleInputEmail1">Delivery Address</label>
+                                        <input class="form-control" v-model="form.address" type="text" placeholder="Enter Address">
+                                    </div>
+                                    <div class="form-group">
+                                        <label for="exampleInputEmail1">Contact #</label>
+                                        <input class="form-control" v-model="form.contact" type="text" name="contact" placeholder="Enter Contact Number">
+                                    </div>
```

Comment:
Will need to show the error text when a user submits an incomplete checkout form